forge/db/migrations/20230328-01-add-userbillingcode-table.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Add UserBillingCode table
+ */
+const { DataTypes } = require('sequelize')
+
+module.exports = {
+    up: async (context) => {
+        await context.createTable('UserBillingCodes', {
+            id: {
```

Comment:
Ideally, this table would have `UserId` as the primary key - as each user can only have one code. But I couldn't get the Sequelize model to create the table in that way due to how it manages associations between models.

Reply:
Right now we have to apply all migrations regardless - we don't have the concept of EE-only migrations. The issue being, a user could be running CE, it applies migrations (skipping this one), they then apply a license key to become EE... but the migration hasn't run.